Source/WebKit/UIProcess/WebPageProxy.h

```diff
@@ -40,6 +40,8 @@
 #include <WebCore/PlatformTextAlternatives.h>
 #endif
 
+#include "WebTextReplacementData.h"
```

Comment:
I think this can be forward declared. We really want to minimize includes in this file.

Reply:
yup, I originally thought it was needed because of the nested enum, but now that I think about it the enum isn’t actually nested